### PR TITLE
Update for code and dependencies

### PR DESCRIPTION
**Pull request by FreedomLabsIO**
Update for code and dependencies for compatibility with latest versions of ArangoDB and MongoDB

### CODE REVIEW

**Comment by FreedomLabsIO** [comment]
@bbonnin

File: src/main/java/io/millesabords/mongo2arango/BulkInsert.java [hunk 1]
```diff
@@ -48,28 +50,31 @@ public void modify(DBCursor cursor) {
...
+			LOGGER.info("ArangoDB | Creating collection : {}", arangoCollection.name());
+			arangoCollection.create();
+		}
+        
         while (cursor.hasNext()) {
             total++;
             count++;
             doc = cursor.next();
-            id = doc.remove("_id").toString();
+            doc.remove("_id").toString();
```
Comment (bbonnin) [inline src/main/java/io/millesabords/mongo2arango/BulkInsert.java:71]: I think you can remove the "toString()" call

**Review by bbonnin — APPROVED** [review]
Thanks @brunoaduarte !!

File: src/main/java/io/millesabords/mongo2arango/BulkInsert.java (hunk 1 above)
Comment (FreedomLabsIO) [reply src/main/java/io/millesabords/mongo2arango/BulkInsert.java:71]: Yes, forgot about that.

Comment (FreedomLabsIO) [reply src/main/java/io/millesabords/mongo2arango/BulkInsert.java:71]: I removed it from my code but i'll wait to submit a new pull request when the features to also convert Arango to Mongo and also check foreign keys are finished.

**Pull request merged**